python/pyspark/pandas/frame.py

```diff
@@ -8844,91 +8836,6 @@ def combine_first(self, other: "DataFrame") -> "DataFrame":
         )
         return DataFrame(internal)
 
-    def append(
```

Comment:
we should not remove these API before the major release Spark 4.0

Reply:
Okay, let me just revert the all the pandas-on-Spark related changes and will address it separately.